conftest.py

```diff
@@ -138,7 +165,9 @@ def _make_valid_request(
 def get_sensor_info(make_valid_request):
     def _get_sensor_info():
         log.info("Get sensor info")
-        return make_valid_request(SensorMethod.GET_INFO)
+        sensor_response = make_valid_request(SensorMethod.GET_INFO)
+        return SensorInfo(**sensor_response)
+        # return make_valid_request(SensorMethod.GET_INFO)
```

Comment:
Better to not have commented code if it's not for explanatory purposes.

Reply:
was left by mistake. fixed